srcpkgs/httpie/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'httpie'
 pkgname=httpie
-version=2.2.0
-revision=2
+version=2.5.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-requests python3-Pygments"
+depends="python3-setuptools python3-requests python3-requests-toolbelt python3-pygments python3-socks python3-defusedxml"
```

Comment:
Please break at ~80 columns.

Reply:
And `pygments` is wrong, `Pygments` was correct.

Reply:
My bad. Thank you @ericonr this is now fixed.